WebPhapp/backend/server.js

```diff
@@ -1,9 +1,34 @@
 const express = require('express');
 const path = require('path');
 var fs = require("fs");
+var conn = require('./connections.js') // private file not under VC.
 
 const app = express();
 
+// establish a connection to the remote MySQL DB
+if(conn.MySQL){
+    var mysql = require('mysql')
+    var connection = mysql.createConnection(conn.MySQL);
+}
+
+// Given a list of drugIDs, looks up drug names in the MySQL DB.
+// Args: drugIDs list <(int or string)>
+// Returns: Promise
```

Comment:
I would also like to see what the promise returns here. I think that would be insightful to know.